src/state/move-to-next-index/in-foreign-list.js

```diff
@@ -1,32 +1,37 @@
 // @flow
 import getDraggablesInsideDroppable from '../get-draggables-inside-droppable';
-import { isPointWithinDroppable } from '../is-within-visible-bounds-of-droppable';
 import { patch } from '../position';
 import moveToEdge from '../move-to-edge';
+import getDisplacement from '../get-displacement';
+import getViewport from '../visibility/get-viewport';
+import isVisibleInNewLocation from './is-visible-in-new-location';
 import type { Edge } from '../move-to-edge';
 import type { Args, Result } from './move-to-next-index-types';
-import type {
+import type
```

Comment:
weird newlines here?

Reply:
Yeah, lame auto import fix. I'll fix this one - but it should probably be an eslint rule eventually

Reply:
fixed